frontend/src/components/eam-create-worker/eam-create-worker.module.scss

```diff
@@ -0,0 +1,46 @@
+.container {
```

Comment:
Style it according to the design
![image](https://user-images.githubusercontent.com/42296182/153875819-1ffaf63e-d267-4c9f-9bae-191f921425e6.png)